assets/js/googlesitekit/notifications/components/layout/NotificationWithSVG.js

```diff
@@ -0,0 +1,98 @@
+/**
+ * Site Kit by Google, Copyright 2024 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * External dependencies
+ */
+import classNames from 'classnames';
+
+/**
+ * Internal dependencies
+ */
+import {
+	BREAKPOINT_SMALL,
+	BREAKPOINT_TABLET,
+	useBreakpoint,
+} from '../../../../hooks/useBreakpoint';
+import { Cell, Grid, Row } from '../../../../material-components';
+
+export default function NotificationWithSVG( {
+	id,
+	title,
+	description,
+	actions,
+	SVG,
```

Comment:
Capital prop names are a bit odd. I'm guessing it's done so you can use `<SVG />` below…

Looks like [we do this elsewhere too](https://github.com/google/site-kit-wp/blob/4e8b12560bfc561dae3d6d7c3419ad2d68a1ba11/assets/js/googlesitekit/notifications/components/layout/NotificationWithSmallSVG.js#L27), so no need to change it, but I guess leaving my comment here so it's recognised 😆

Reply:
Hmm, this just clearly screams that we need a component as a prop here - similar to how we use pass `Widget`, `WidgetNull`, `Notification`, etc as well.